protocols/v2/framing-sv2/src/framing.rs

```diff
@@ -57,23 +55,23 @@ impl<T: Serialize + GetSize, B: AsMut<[u8]> + AsRef<[u8]>> Sv2Frame<T, B> {
     /// When called on a non serialized frame, it is not so cheap (because it serializes it).
```

Comment:
we should update these comments so that they are coherent with the new behavior of this function

updating the comments is especially important given that these are used on the Rust Docs, so they need to follow these changes accordingly